assignments/callbacks.js

```diff
@@ -27,29 +27,52 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  const arrLength = items.length;
+  console.log(arrLength);
```

Comment:
We want to pass `arrLength` to `cb`, not `console.log`. So it should instead by `cb(arr.Length)`. That we can pass `console.log` as our callback if we want, or anything else.